### PR TITLE
Issue66 - Add CI parameter

### PR DESCRIPTION
**Pull request by Biboba**
If user specify --ci parameter, then no "[ci skip] ***NO_CI***" is not included in commit message

It closes https://github.com/leonardoanalista/corp-semantic-release/issues/66

### CODE REVIEW

File: src/lib/addFilesAndCreateTag.js [hunk 1]
```diff
@@ -3,15 +3,21 @@ const terminateProcess = require('./helpers').terminateProcess;
...
   let code;
   // ###### Add edited files to git #####
   log.info('>>> About to add and commit package.json and CHANGELOG...');
   code = shell.exec('git add package.json CHANGELOG.md').code;
   terminateProcess(code);
 
   // ###### Commit files #####
-  code = shell.exec('git commit -m "chore(release): ' + newVersion + ' [ci skip] ***NO_CI***"').code;
+  let commitMessage = 'git commit -m "chore(release): ' + newVersion;
+  if (!ci) {
```
Comment (leonardoanalista) [inline src/lib/addFilesAndCreateTag.js]: Hi @Biboba Thanks for the PR first of all. I have a few questions here:

* `semantic-release` has option `ci` default set to `true` - it means it only adds `[ci skip]` if we set it to `--ci`.  In your case you have the opposite condition here. 
* I also find very confusing the short name `ci`. I know we agreed to this variable name but what do you think if we use `ci-skip` or `no-ci`? Second mirrors semantic-release api. 
* could we make it default `true`, like semantic-release. If so, we need to make it a `Breaking Change` in this version and analyze the impact of the new version. 
* Would you mind updating the README with the new version? I know it's a duplication of the `program.options`. I am open for suggestions here too to have only one place where we modify the options and keep readme updated. 

refs:
https://github.com/semantic-release/semantic-release/blob/caribou/docs/usage/configuration.md#ci

**Comment by leonardoanalista** [comment]
Hi @uglow how are you? If you still use this tool, feel free to give your opinion too.

**Comment by uglow** [comment]
@leonardoanalista - I’m good thanks, maybe hit me up on LinkedIn for a chat. 

@Biboba, nice PR 👍🏻 
I think it could be improved if it follows the conventions @leonardoanalista referred to above.

**Comment by Biboba** [comment]
Hello,

Happy to make the changes.

But I had a closer look to semantic-release package and this --ci option is not exactly what I thought it is : it does not involve any "[ci skip] _**NO_CI**_" message.
It is rather an option to be able to make a local release without using any CI (Travis, Gitlab,...). It is basically what this plugin does.

Apologize for the confusion but still I think it would be interesting to be able to specify "NO_CI" as an option. I don't know if it changes the way you feel about this PR.

Thanks for the feedback !

References:

- https://github.com/semantic-release/semantic-release/blob/master/docs/support/FAQ.md#can-i-run-semantic-release-on-my-local-machine-rather-than-on-a-ci-server
- https://github.com/semantic-release/semantic-release/blob/aed4ea2edc1bf6d4e452a6cea421899aa497016f/test/index.test.js#L635-L694

**Comment by leonardoanalista** [comment]
Thanks @Biboba for your research. Considering we are talking about different options between semantic-release and this package, we could do whatever we want here. 

* We can avoid the breaking change. In this case, we could call this option `ci-skip` instead of `ci`
* Make `ci-skip` default false to avoid the breaking change. 
* Option `ci-skip` will allow local release as every CI skips the build if commit message contains `[ci skip]`
* Update read me to have the new option: https://github.com/leonardoanalista/corp-semantic-release#options

What do you think about the points above?

**Comment by leonardoanalista** [comment]
addressed by https://github.com/leonardoanalista/corp-semantic-release/pull/71

**Pull request closed without merging**